pkg/controller/statefulset/stateful_set_utils.go

```diff
@@ -377,13 +377,12 @@ func inconsistentStatus(set *apps.StatefulSet, status *apps.StatefulSetStatus) b
 		status.UpdateRevision != set.Status.UpdateRevision
 }
 
-// completeRollingUpdate completes a rolling update when all of set's replica Pods have been updated
+// completeUpdate completes an update when all of set's replica Pods have been updated
 // to the updateRevision. status's currentRevision is set to updateRevision and its' updateRevision
 // is set to the empty string. status's currentReplicas is set to updateReplicas and its updateReplicas
```

Comment:
the second part of this comment about clearing the values does not seem to hold true

Reply:
please keep this part there: `status's currentReplicas is set to updateReplicas`